### ISSUE TITLE
Why pre-initialize esbuild-svelte with null?

### ISSUE THREAD

**Issue by jakobrosenberg** [EMH333/esbuild-svelte#12]
I was going to submit a PR for this, but it seems that the decision to initialize `esbuild-svelte` with `null` is deliberate.

**Comment by EMH333** [comment]
Good question... Essentially all the `null` does right now is initialize everything with defaults. The idea was that someone could do something like `import esbuildSvelte from 'esbuild-svelte'` and treat `esbuildSvelte` as a variable instead of a function. Essentially trying to keep things as "simple" as possible.

However, as I am looking at what other plugins are doing, and how much confusion this actually ends up adding, I will probably remove that functionality in v1 so there will be one import in the future.

Essentially
```javascript
import sveltePlugin from 'esbuild-svelte';
import { build } from 'esbuild';

build({
  ...
  plugins: [sveltePlugin()]
});
```
for simple cases and
```javascript
build({
  ...
  plugins: [sveltePlugin({
    compileOptions: {
      legacy: true,
      dev: true,
    },
    preprocessor: anExamplePreprocessor
  })]
});
```
 for more complex cases will work from the same import.

This will be a breaking change, but one that makes a lot of sense in my mind. I haven't had the time to move everything over yet. I've been watching the esbuild repo and will probably to the update after Evan Wallace releases the next evolution of changes to the plugin api.

Wow, that was an extended answer to a simple question, sorry about that :)

TL;DR: Cluelessness on my part for the best way to keep things simple

**Comment by EMH333** [comment]
FYI, I've released `v0.4.0` which unites everything under one import (which then must be called a function, regardless of advanced options or not). Hopefully this \*actually\* simplifies things?

I'll close this issue b/c it is no longer relevant to the current code state, please re-open if I'm missing something
